include/SampleBuffer.h

```diff
@@ -109,6 +109,10 @@ class LMMS_EXPORT SampleBuffer : public QObject, public sharedObject
 	SampleBuffer( const QString & _audio_file, bool _is_base64_data = false );
 	SampleBuffer( const sampleFrame * _data, const f_cnt_t _frames );
 	explicit SampleBuffer( const f_cnt_t _frames );
+	SampleBuffer( SampleBuffer& orig );
+
+	friend void swap( SampleBuffer& first, SampleBuffer& second ) noexcept;
+	SampleBuffer& operator= ( SampleBuffer& that );
```

Comment:
I left it that way to match the surrounding code. Otherwise I would either inflate the diff with unrelated changes or have `name(args)` immediately below `name( args )`.

Reply:
We always hit that dead end with code style. Wasn't the agreement that every line touched by the diff (either added or edited) should be compliant to the guidelines, independent of the surrounding code? Whatever the rule is, that's something we have to make clear, as in "included in the guidelines" clear :grimacing:

Reply:
It won't be an issue once the auto reformat is done, because at that point matching the surrounding code and matching the style guidelines won't conflict. Therefore I don't think it's something we need to put in the style guideline.